lib/src/runner/configuration/load.dart

```diff
@@ -289,6 +266,32 @@ class _ConfigurationLoader {
         excludeTags: excludeTags);
   }
 
+  Map _loadFoldedStackFrames() {
+    bool foldOptionSet = false;
+    return _getMap("fold_stack_frames", key: (keyNode) {
+      _validate(keyNode, "Must be a string", (value) => value is String);
+      if (keyNode.value != "only" && keyNode.value != "except") {
+        throw new SourceSpanFormatException(
+            'Must be "only" or "except".', keyNode.span, _source);
+      }
```

Comment:
Any statement of the form `if (something about value) { throw }` should probably be a call to `_validate()` instead.